docs/how-to/manage-data/local-facts-uploader.mdx

```diff
@@ -1,282 +1,352 @@
 ---
 sidebar_position: 3
-title: Send Critical Updates
+title: Proxy Facts via PDP
```

Comment:
I really think we should stick with critical updates as a name for this.
Focusing on what it means for the customer, and less of what it technically does...

Reply:
But it's not the name of the feature in the code, and "Critical Updates" meaning is vague and increases the confusion further